dali/base/dafdesc.cpp

```diff
@@ -1451,6 +1468,9 @@ class CFileDescriptor:  public CFileDescriptorBase, implements ISuperFileDescrip
             attr.setown(createPTreeFromIPT(at));
         else
             attr.setown(createPTree("Attr"));
+
+        extendedVersion = serializationVersion;
```

Comment:
Normally you'd use the serialization version to validate during deserialization only, you wouldn't need or store it beyond that, so I'm not sure it needs to be a class member like this.